assets/karma.conf.js

```diff
@@ -25,20 +28,19 @@ module.exports = function (config) {
             'src/scripts/**/*.js'
         ],
 
-
         // list of files / patterns to exclude
         exclude: [
             'src/scripts/index.js'
         ],
 
-
         // preprocess matching files before serving them to the browser
         // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
         preprocessors: {
             'src/scripts/**/*.js': ['browserify']
         },
+
         browserify: {
-            debug: true,
+            debug: process.argv.some(isDebug),
```

Comment:
So how do we run with debug now? I wonder if we can add it to the scripts, at least test:watch.

Reply:
I didn't intend to change behavior, but to condense the logic in this section: https://github.com/usgs/waterdataui/blob/master/assets/karma.conf.js#L299-L309

... which was I believe overwriting the debug value set here.